### DISCUSSION TITLE
Request for Help Reading H5 File

### DISCUSSION THREAD

**Discussion by Roaming-Mars** [snl-dakota/dakota#77] [Q&A]
I would like to read the correlation results (example: simple/simple ranked) from the output H5 file into python for visualization, but cannot seem to find the values listed. Is there any documentation listing the H5 structure? Additionally, are there any scaling factors that must be applied? Thank you!

**Comment by jadamstephens** [comment]
The structure is documented [here](https://snl-dakota.github.io/docs/6.19.0/users/usingdakota/output/hdf.html).

To save you some time, the results are most likely at `/methods/NO_METHOD_ID/results/execution:1/`. If you named the method block using the `id_method` keyword, then `NO_METHOD_ID` will be replaced by that name.

**Comment by Roaming-Mars** [comment]
Thank you for the documentation! Unfortunately, I am having trouble understanding the description for how to extract the correlation results (simple/partial/ranked). Are there any examples for this scripting? Thank you!
